pom.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>2.2.2.RELEASE</version>
+		<relativePath /> <!-- lookup parent from repository -->
+	</parent>
+	<groupId>com.github.virtualgamers</groupId>
+	<artifactId>hidden-role-to-go-rest</artifactId>
+	<version>0.1.0-SNAPSHOT</version>
+	<name>hidden-role-to-go-rest</name>
+	<description>Hidden Role To Go Rest Application</description>
+
+	<properties>
+		<java.version>1.8</java.version>
+	</properties>
+
+	<dependencies>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-amqp</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-web</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework</groupId>
+			<artifactId>spring-webmvc</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>com.github.virtualgamers</groupId>
+			<artifactId>hidden-role-to-go-core</artifactId>
+			<version>0.1.0-SNAPSHOT</version>
```

Comment:
will this be the same as the project.version?
if not, can we at least define it in dependencyManagement or as a tag at the top or something?

Reply:
I'm going to say tag. No reason to create a DependencyManagement section for a single pom project.

Reply:
By that I mean there is no reason to duplicate all the dependencies for a single pom file.

Reply:
👍 tag is fine